### PR TITLE
Fix linting rule in pyproject.toml

### PR DESCRIPTION
**Pull request by Sahel13**
See https://github.com/state-space-models/cuthbert/pull/240#issuecomment-4606080492

### CODE REVIEW

File: tests/cuthbertlib/resampling/test_resamplings.py [hunk 1]
```diff
@@ -154,8 +154,5 @@ def resampled_lse(sigma, resampling_fn):
         grad_stop = jax.grad(
             lambda sigma: resampled_lse(sigma, stop_gradient_decorator(resampling_fn))
         )(true_sigma)
-        grad_plain = jax.grad(lambda sigma: resampled_lse(sigma, resampling_fn))(
```
Comment (SamDuffield) [inline tests/cuthbertlib/resampling/test_resamplings.py:157]: why did this get deleted?

Comment (Sahel13) [reply tests/cuthbertlib/resampling/test_resamplings.py:157]: The test was not using it (somehow i started a review with my previous reply XD)

Comment (SamDuffield) [reply tests/cuthbertlib/resampling/test_resamplings.py:157]: Maybe it should be? haha

@DanWaxman @AdrienCorenflos ?

Comment (DanWaxman) [reply tests/cuthbertlib/resampling/test_resamplings.py:157]: Oh, hmm -- I think it's fine. The test currently asserts that the stop-gradient-derived gradient (`grad_stop`) is close to the analytical gradient (`grad_base`). The unused `grad_plan` is just the un-corrected gradient, and should be identically zero, but this isn't really in scope for `test_stop_gradient_resampling`.

**Review by SamDuffield — APPROVED** [review]
